Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Converters/FIR-Weights.cpp

```diff
@@ -297,7 +297,6 @@ namespace ImageProcessingAtom
                 if (trimZeros)
                 {
                     /* set i0 and i1 to the nonzero support of the filter */
-                    i0 = i0;
                     i1 = i1 = lastnonzero + 1;
                 }
```

Comment:
maybe this was meant to read:
```cpp
i0 = i1 = lastnonzero+1;
```

Reply:
yeah, I was struggling with that... in general I dont like changing behavior when doing these things... 
seems reasonable to do that, will change and see if something fails...

Reply:
actually, this produces asset processing errors, so leaving it without behavior change